.buildbot.sh

```diff
@@ -21,6 +21,12 @@ make -j `nproc` install
 make -j `nproc` check-all
 cd ..
 
+# clang-format any new files that we've introduced ourselves.
+PATH=${INST_DIR}/bin:${PATH}
+clang-format -i llvm/lib/Transforms/Yk/ControlPoint.cpp \
```

Comment:
I think we'll forget to add file names here. How about automatically checking the formatting of any files that don't exist on the main llvm fork? According to https://stackoverflow.com/questions/19222708/git-how-to-list-only-newly-added-files-between-two-branches we should be able to do something like:

```
git diff --name-only --diff-filter=A llvm-project/master `git branch --show-current` | xargs clang-format -i
```